ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -70,31 +70,19 @@ export default class ConfirmTransactionBase extends Component {
     actionKey: PropTypes.string,
     contentComponent: PropTypes.node,
     dataComponent: PropTypes.node,
-    detailsComponent: PropTypes.node,
-    errorKey: PropTypes.string,
-    errorMessage: PropTypes.string,
     primaryTotalTextOverride: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.node,
     ]),
     secondaryTotalTextOverride: PropTypes.string,
     hideData: PropTypes.bool,
-    hideDetails: PropTypes.bool,
     hideSubtitle: PropTypes.bool,
     identiconAddress: PropTypes.string,
-    onCancel: PropTypes.func,
     onEdit: PropTypes.func,
-    onEditGas: PropTypes.func,
-    onSubmit: PropTypes.func,
     setMetaMetricsSendCount: PropTypes.func,
     metaMetricsSendCount: PropTypes.number,
-    subtitle: PropTypes.string,
     subtitleComponent: PropTypes.node,
-    summaryComponent: PropTypes.node,
     title: PropTypes.string,
-    titleComponent: PropTypes.node,
-    valid: PropTypes.bool,
-    warning: PropTypes.string,
     advancedInlineGasShown: PropTypes.bool,
     insufficientBalance: PropTypes.bool,
     hideFiatConversion: PropTypes.bool,
```

Comment:
changes to here in this file look good. I verified that none of these properties are passed to any of the instances of ConfrimTransactionBaseComponent